routes/Favorite.js

```diff
@@ -42,4 +42,37 @@ router.post('/addFav', urlencodedParser, async(req, res) => {
     }
 });
 
+router.post('/removeFav', urlencodedParser, async(req, res) => {
+    let username = await favorite.Favorite.findOne({
+        username: req.body.username
+    });
+
+    if (!username) {
+        res.status(400).send('No celebrities in the list to remove');
+    } else {
+        // Check if selected celebrity is in the favorite list
+        var remove = false;
+        username.favorite.forEach(function(item, index) {
```

Comment:
`arr.includes(obj);`

Reply:
what do you mean by arr.inclues(obj)? what does it supposed to do?